doc/user_guide/index.rst

```diff
@@ -66,9 +66,6 @@ concepts in HoloViews:
 `Working with large data <Large_Data.html>`_
  Leverage Datashader to interactively explore millions or billions of datapoints.
 
-`Working with Streaming Data <Streaming_Data.html>`_
- Demonstrates how to leverage the streamz library with HoloViews to work with streaming datasets.
```

Comment:
Maybe this is too aggressive. We can just let it be for now.

Reply:
Why removing the ling to this notebook, it's still available right?